src/caret_analyze/architecture/architecture.py

```diff
@@ -36,22 +36,31 @@
 
 logger = logging.getLogger(__name__)
 
+MAX_CALLBACK_CONSTRUCTION_ORDER_ON_PATH_SEARCHING = 10
```

Comment:
I suggest changing variable name to `DEFAULT_MAX_CALLBACK_CONSTRUCTION_ORDER_ON_PATH_SEARCHING`. (adding `DEFAULT_`)

Reply:
I agree.